tests/test_cache.py

```diff
@@ -0,0 +1,147 @@
+from django.template import Context
+
+from cms.api import add_plugin
+from cms.cache.placeholder import get_placeholder_cache
+from cms.test_utils.util.fuzzy_int import FuzzyInt
+
+from djangocms_alias.cms_plugins import Alias
+
+from .base import BaseAliasPluginTestCase
+
+
+class AliasCacheTestCase(BaseAliasPluginTestCase):
+    alias_template = """{% load djangocms_alias_tags %}{% render_alias plugin.alias %}"""
+
+    def setUp(self):
+        super().setUp()
+        # Create an alias and get a placeholder
+        self.alias = self._create_alias(published=True)
+        self.alias_placeholder = self.alias.get_placeholder(self.language)
+        # Create page and add alias
+        self.page = self._create_page('test')
+        self.page_placeholder = self.page.get_placeholders(self.language).get(slot='content')
+
+    def test_create_published_alias(self):
+        """
+        A published page populated with placeholders and plugins
+        should contain the content it is populated with when rendered
+        """
+        add_plugin(
+            self.alias_placeholder,
+            'TextPlugin',
+            language=self.language,
+            body='Content Alias 1234',
+        )
+        # Create alias plugin inside the page placeholder
+        add_plugin(
+            self.page_placeholder,
+            Alias,
+            language=self.language,
+            alias=self.alias,
+        )
+
+        # Get page via http request
+        with self.login_user_context(self.superuser):
+            response = self.client.get(self.page.get_absolute_url(self.language))
+
+        # Check the response contains the content we added to our page
+        self.assertContains(response, 'Content Alias 1234')
+
+    def test_query_plugin_count(self):
+        """
+        Aliases are cached, so we should see a decrease in the number of queries
```

Comment:
The test doesn't accomplish what the description says it should. There are no checks on queries ran in this test. The test needs logic added to prove the number of queries are lower.